.github/workflows/pr-open.yml

```diff
@@ -101,6 +101,7 @@ jobs:
               -p URL=${{ needs.prep.outputs.url }} -p CERTBOT=false -p REPLICA_COUNT=1
               -p FOM_EMAIL_NOTIFY=SIBIFSAF@victoria1.gov.bc.ca -p SITEMINDER_URL="https://logontest7.gov.bc.ca"
```

Comment:
remove siteminder URL?

Reply:
A ticket to clean up is at the backlog. Intended to remove it altogether.